spec/controllers/instruments_controller_spec.rb

```diff
@@ -276,6 +276,10 @@
                                       type: RelaySynaccessRevA.name,
                                       auto_logout: true,
                                       auto_logout_minutes: 15,
```

Comment:
I think the relay related logic in this file should be moved into the new system spec, since it is describing a code path that won't be used any more